services/replayer/call_tracer_collector.go

```diff
@@ -17,11 +18,14 @@ const (
 	TracerName   = "callTracer"
 )
 
-func DefaultCallTracer() (*tracers.Tracer, error) {
+func DefaultCallTracer(chainID *big.Int) (*tracers.Tracer, error) {
```

Comment:
Maybe lets call this `evmChainID` to avoid confusion.

another alternative would be to introduce a type:
`type EVMChainID = *big.Int`